CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+# Contributing Guide
+
+* [New Contributor Guide](#contributing-guide)
+  * [Ways to Contribute](#ways-to-contribute)
+  * [Find an Issue](#find-an-issue)
+  * [Ask for Help](#ask-for-help)
+  * [Pull Request Lifecycle](#pull-request-lifecycle)
+  * [Development Environment Setup](#development-environment-setup)
+  * [Sign Your Commits](#sign-your-commits)
+  * [Pull Request Checklist](#pull-request-checklist)
+
+Welcome! We are glad that you want to contribute to our project! 💖
+
+As you get started, you are in the best position to give us feedback on areas of
+our project that we need help with including:
+
+* Problems found during setting up a new developer environment
+* Gaps in our Quickstart Guide or documentation
+* Bugs in our automation scripts
+
+If anything doesn't make sense, or doesn't work when you run it, please open a
+bug report and let us know!
+
+## Ways to Contribute
+
+We welcome many different types of contributions including:
+
+* New features
+* Builds, CI/CD
+* Bug fixes
+* Documentation
+* Issue Triage
+* Answering questions on Github issues
+* Communications / Social Media / Blog Posts
+* Release management
+
+## Find an Issue
+
+We have good first issues for new contributors and help wanted issues suitable
+for any contributor. [good first issue](https://github.com/coralogix/telemetry-shippers/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) has extra information to
+help you make your first contribution. [help wanted](https://github.com/coralogix/telemetry-shippers/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) are issues
+suitable for someone who isn't a core maintainer and is good to move onto after
+your first pull request.
+
+Once you see an issue that you'd like to work on, please post a comment saying
+that you want to work on it. Something like "I want to work on this" is fine.
+
+## Ask for Help
+
+The best way to reach us with a question when contributing is to ask on the original github issue.
+
+## Pull Request Lifecycle
+
+- Open a PR; if it's not finished yet, please make it a draft first
+- Reviewers will be automatically assigned based on code owners
+- Reviewers will get to the PR as soon as possible, but usually within 2 days
+
+## Development Environment Setup
+
+You must have a Helm Chart CLI available in your environment to install the charts on this repository.
+
+If you don't have Helm installed yet, please check the official Helm documentation [here](https://helm.sh/docs/helm/helm_install/).
+
+## Sign Your Commits
+
+### DCO
```

Comment:
Wait now I'm confused, I though we use CLA, not DCO? I think we need to require only one.

Reply:
Should I keep only CLA so?

Reply:
Looks like we do have DCO though, I'm confused 🤔 

![image](https://github.com/coralogix/telemetry-shippers/assets/24193764/ea92c6f9-94e1-4543-8ed1-cc28c1658db6)

Reply:
DCO was added by me by mistake!
I'm removing the bot and also removing from document.